templates/common/on-prem/files/keepalived.yaml

```diff
@@ -97,7 +97,19 @@ contents:
             done
           }
 
+          remove_vip()
+          {
+            address=$1
+            interface=$(ip -o a | grep $address | awk '{print $2}')
+            if [ -n "$interface" ]; then
+                ip a del $address dev $interface
```

Comment:
Shouldn't you specify also the netmask here?

I checked it in my environment and I got [1] warning 

[1] 

+ remove_vip 192.168.111.5
+ address=192.168.111.5
++ ip -o a
++ grep 192.168.111.5
++ awk '{print $2}'
+ interface=enp2s0
+ '[' -n enp2s0 ']'
+ ip a del 192.168.111.5 dev enp2s0
Warning: Executing wildcard deletion to stay compatible with old scripts.
         Explicitly specify the prefix length (192.168.111.5/32) to avoid this warning.
         This special behaviour is likely to disappear in further releases,
         fix your scripts!

Reply:
Yes, you're right. I had mistakenly hard-coded the mask in my first version of this patch, and didn't notice that it triggered the warning after I removed it. The latest version should handle it correctly.